### ISSUE TITLE
[DesignerBug 31602479: [Functional - Adaptive Cards - Inputs.Json]: The functionality of radio buttons are not working as expected in 'Input.Text element' Pane.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7630]
### Problem Description

[Bug 31602479](https://microsoft.visualstudio.com/OS/_workitems/edit/31602479): [Functional - Adaptive Cards - Inputs.Json]: The functionality of radio buttons are not working as expected in 'Input.Text element' Pane.

User Experience:

It will impact the all the users if the functionality of the radio button is improper as the buttons are getting selected by using space key.

Test Environment:

Application: Adaptive Cards
OS version: 2004 (OS Build 21301.1000)

Application Version: 2101.28.1.0

Pre-Requisite: Launch the Adaptive Card application.

Repro steps:

1. Launch the Adaptive Card application.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to Inputs.Json button and activate it.

4. Adaptive card 'Input.Text element'  will be appeared on right sight of the pane.

5. Navigate to Red radio button and verify the issue. 

Actual Result:

The functionality of radio buttons are not working as expected in 'Input.Text element' Pane. The radio buttons are getting selected by pressing space key on radio buttons.

Note:

This issue is repro'ing thorough out the application

Expected result:

The radio buttons should be getting selected automatically when focus lands on it by using up and down arrow key.

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vaishali1397** [comment]
Issue has been fixed in below test environment, hence adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25330.1000)
Application: AdaptiveCardVisualizer

**PFA For Reference:** Adaptive Card UWP~ Issue fixed.mp4

https://user-images.githubusercontent.com/104483005/235417214-62679d63-fac4-4ee2-9bd4-e8d0b039d570.mp4
